flashinfer/gdn_kernels/blackwell_prefill/gdn_helpers.py

```
@@ -0,0 +1,190 @@
+    :type kind:          SmemLayoutAtomKind
+
+    :return: SMEM layout for tensor A
+    :rtype: Union[cute.Layout, cute.ComposedLayout]
+    """
+
+    # is_k_major = tiled_mma.op.a_major_mode == cutlass.cute.nvgpu.common.OperandMajorMode.K # internal 
+    is_k_major = tiled_mma.op.a_major_mode == cutlass.cute.nvgpu.tcgen05.OperandMajorMode.K
+    a_smem_shape = tiled_mma.partition_shape_A(
+        cute.dice(mma_tiler_mnk, (1, None, 1), loc=loc, ip=ip)
+    )
+    a_smem_shape_mn_k = (
```

Ruff (0.15.5)

[error] Local variable `a_smem_shape_mn_k` is assigned to but never used

Remove assignment to unused variable `a_smem_shape_mn_k`

(F841)